<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mqColorDialog</class>
 <widget class="QDialog" name="mqColorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color and lighting options</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="mqColorChooserButton" name="meshColorButton">
        <property name="text">
         <string>Default solid color of surfaces</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="mqColorChooserButton" name="backgroundColorButton">
        <property name="text">
         <string>Lower background color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="mqColorChooserButton" name="backgroundColorButton2">
        <property name="text">
         <string>Upper background color</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Ambient lighting :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="ambient">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Diffuse lighting   :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="diffuse">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Specular lighting  :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="specular">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Specular power</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="specularPower"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="preset0">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Icons/icons.qrc">
          <normaloff>:/Icons/preset00.png</normaloff>:/Icons/preset00.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="preset1">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Icons/icons.qrc">
          <normaloff>:/Icons/preset01.png</normaloff>:/Icons/preset01.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="preset2">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Icons/icons.qrc">
          <normaloff>:/Icons/preset02.png</normaloff>:/Icons/preset02.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="preset3">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Icons/icons.qrc">
          <normaloff>:/Icons/preset03.png</normaloff>:/Icons/preset03.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>239</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="reinitcolorsButton">
      <property name="text">
       <string>Reinitialize</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mqColorChooserButton</class>
   <extends>QToolButton</extends>
   <header location="global">mqColorChooserButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mqColorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mqColorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
